app/views/equipment_models/index.html.erb

```diff
@@ -4,8 +4,12 @@
   <thead>
   <tr>
     <th>Name</th>
-    <%= "<th>Type</th>".html_safe unless @category %>
-  <%# these blank TH's necessary for the sort function %>
+    <th class="no_sort">Current Status</th>
```

Comment:
I'm not sure why we weren't already doing this, but shouldn't we be using the `_table.html.erb` partial here?